src/components/SingleArticleCard.jsx

```diff
@@ -56,7 +92,10 @@ function SingleArticleCard() {
                 <li className="each-comment" key={comment.comment_id}>
                   <p>Author: {comment.author}</p>
                   <p>Comment: {comment.body}</p>
-                  <p>Created At: {comment.created_at}</p>
+                  <p>
+                    Created At:{" "}
+                    {dayjs(comment.created_at).format("HH:mm:ss - DD-MM-YYYY")}
+                  </p>
                   <p>Votes: {comment.votes}</p>
                   <br />
                 </li>
```

Comment:
A side note - your Home and Article button links are nearly invisible once they have been clicked, they go a quite similar purple to your background
You can alter the css of a clicked link using a:visited css object